scripts/gha/build_testapps.py

```diff
@@ -195,15 +184,17 @@ def main(argv):
   else:
     timestamp = ""
 
+  config = config_reader.read_config()
+
+  if _IOS in platforms:
+    _build_ios_framework_from_repo(FLAGS.sdk_dir, testapps, config)
```

Comment:
If any framework fails to build, the entire script will stop, which is not what we want: errors in iOS should not affect desktop or Android, and a failure for database should not affect auth. 

It would be better if the frameworks needed for a given API were built within the try block (line 268) for the iOS build of that platform.